### PR TITLE
fix: fixup changelog changes and add missing ones

### PR DESCRIPTION
**Pull request by dune73**
I forgot to merge changelog PR #3472 leading to merge conflict and #3474 used wrong verb forms.

This fixes both.

@theseion can you cast a glimpse?

This change does not have to land in the changelog itself.

### CODE REVIEW

File: .changes-pending.md [hunk 1]
```diff
@@ -9,8 +9,13 @@
...
 * fix: improve rule by matching non-word-boundary of commands with options (932237 PL3) (Max Leske) [#3425]
 * fix: replace backend docker container for tests to fix JSON Unicode reflection (Max Leske) [#3464]
-* fix: Unwanted match of character set substrings in multipart/form-data requests (922100 PL1) (Jozef Sudolský) [#3470]
-* fix: FPs with word "settings" (932236 PL2, 932237 PL3, 932239 PL2) (@EsadCetiner) [#3394]
+* feat: add .vscode to restricted-files.data (930130 PL1) (Frederik Himpe) [#3471]
+* feat: remove redundant t:lowercase for a little performance (922110 PL1) (Jozef Sudolský) [#3469]
+* fix: update comments (922110 PL1, 942440 PL2) (Jozef Sudolský) [#3468]
+* feat: add unix commands pyversions and py3versions (932235 PL1, 932260 PL1, 932236 PL2, 932237 PL3, 932239 PL2) (Jitendra Patro) [#3465]
+* feat: add new test method: check for tags on rules against allowlist (Ervin Hegedus) [#3437]
+* fix: refine unwanted match of character set substrings in multipart/form-data requests (922100 PL1) (Jozef Sudolský) [#3470]
```
Comment (theseion) [inline .changes-pending.md:20]:
```suggestion
* fix: unwanted match of character set substrings in multipart/form-data requests (922100 PL1) (Jozef Sudolský) [#3470]
```

**Review by theseion — CHANGES_REQUESTED** [review]

File: .changes-pending.md (hunk 1 above)
Comment (dune73) [reply .changes-pending.md:20]: I try to have a "verb" at the beginning of every entry. This was the best one I could find.

You insist we should avoid it here?

Comment (theseion) [reply .changes-pending.md:20]: There you go :)

Comment (dune73) [reply .changes-pending.md]: Brillant

**Comment by dune73** [comment]
FCK, I need you to approve the change again @theseion. Please do so tonight, or we have new stuff piling up.

**Review by theseion — APPROVED** [review]

**Pull request merged**